### PR TITLE
Fix buy button not render properly

### PR DESCRIPTION
**Pull request by JuanAlmeida12**
#### What is the purpose of this pull request?
Fix buy button not render properly

#### What problem is this solving?
Buy button render only on hover.

#### How should this be manually tested?
[Access this workspace](https://fixbuybutton--storecomponents.myvtex.com/)

#### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/index.js [hunk 1]
```diff
@@ -223,6 +223,12 @@ class ProductSummary extends Component {
       'bb b--muted-4 ma2': showBorders
     })
 
+    const getBuyButtonVisibility = () => {
+      return !((showButtonOnHover || showBuyButton) && showBuyButton) ? 'hidden' : 'visible'
```
Comment (luciannojunior) [inline react/index.js]: `showButtonOnHover` does not matter on this clause. The visibility only depends on `showBuyButton` in this case. If `true`, 'visible, if not `hidden`.

What's the expected behavior?

File: react/index.js [hunk 2]
```diff
@@ -223,6 +223,12 @@ class ProductSummary extends Component {
...
+    }
+
+    const style = { visibility: getBuyButtonVisibility() }
```
Comment (luciannojunior) [inline react/index.js]: Use `tachyons`'s classes instead. `classname` could do it!

Comment (JuanAlmeida12) [reply react/index.js]: `tachyons` don't support change the `visibility` attribute.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Fix how buy button is rendered
```
Comment (brunojdo) [inline CHANGELOG.md]: Change to: "Now, buy button doesn't move the product summary box when is hidden."

**Pull request merged**